rollup.config.js

```diff
@@ -0,0 +1,47 @@
+import replace from 'rollup-plugin-replace';
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import {uglify} from 'rollup-plugin-uglify';
+import babel from 'rollup-plugin-babel';
+
+function createConfig(file, opts = {}) {
+  const options = Object.assign({
+    production: false,
+    uglify: false
+  }, opts);
+
+  const config = {
+    input: 'index.js',
+    output: {
+      file: file,
+      format: 'umd',
+      name: 'PropTypes'
+    },
+    plugins: [
+      replace(stripEnvVariables(options.production)),
+      nodeResolve(),
+      commonjs(),
+      babel({
+        exclude: 'node_modules/**'
+      })
+    ]
```

Comment:
The order of plugins matter. commonjs and replace plugins are able to consume only stable spec syntax. Better to keep babel before others.